### ISSUE TITLE
Nothing's happening!

### ISSUE THREAD

**Issue by skvaller** [inkstitch/inkstitch#1438]
From start a lot of issues. The software is very sensitive and won't run/render after fix or restart. Have the latest version installed on win10. Been trying different solutions for 48 hours, including rendering 1 color single objects. No problems found, but also no params and no render.

You got any cool ideas I can try before I give up?

**Comment by kaalleen** [comment]
Can you describe what exactly is happening when you click on params?

**Comment by skvaller** [comment]
Well from the start I could do one color, so my installation worked and it was easy enough. Then I was taking on the logotype which were the reason for the extension. I had slimmed it down to 9 colors, each one in a different layer. 
1. I params
2. I got the red and the yellow warnings, cleaned the images up, through the two suggestions until it came back OK
3. I marked my layers and ran params again.
4. Nothing happens...
5. I change to visual
6. The visual keeps loading forever
7. I tried to reinstall, same -nothing happens
8. I try each layer by itself, nothing happens.

**Comment by skvaller** [comment]
Don't know if this is an issue which van be connected:
Inkscape tried to pull files or something from the "Ultra Office" software but was denied. Little bit peculiar perhaps?

**Comment by Tina-Chris** [comment]
Did you run the troubleshooting function before running visuals? Do this for each layer. 
Often it points to nodes which cannot be interpreted.
Run it until the function says "no error detected". (or similar).
Hope this helps
Tina

**Comment by krolco** [comment]
@skvaller Inkstitch 2.0 is written in Python, which is packaged into an executable. When I ran Inkstitch 2.0 on Windows 10, Windows Defender quietly flagged one of the executables as malware (a false positive) and removed the executable from the  Inkstitch directory. The behavior I got sounds a lot like what you're describing. There's a bunch of variations of this (see also https://github.com/inkstitch/inkstitch/issues/710) I [bravely|stupidly] worked around it by telling Windows Defender to ignore my Inkstitch folder. That might help.
